salt/cli/salt.py

```diff
@@ -13,14 +12,18 @@
 from salt.utils.args import yamlify_arg
 from salt.utils.verify import verify_log
 from salt.exceptions import (
-        SaltClientError,
-        SaltInvocationError,
-        EauthAuthenticationError
-        )
+    EauthAuthenticationError,
+    LoaderError,
+    SaltClientError,
+    SaltInvocationError,
+    SaltSystemExit
+)
 
 # Import 3rd-party libs
 from salt.ext import six
 
+sys.modules['pkg_resources'] = None
```

Comment:
IIRC, this needed to be at the very top of this file to work around a serious performance issue. Is there a reason it was moved?

Reply:
It looked very out of place at the top of the imports. I wondered if there was a reason it was up there, but there isn't a comment about why it was there.

Reply:
Yes, I should have documented that better. If you could please move it back there, I will do that.

Reply:
Sounds good. All fixed. :)